SimPEG/electromagnetics/static/utils/static_utils.py

```diff
@@ -141,12 +143,12 @@ def electrode_separations(survey_object, electrode_pair="all", **kwargs):
             B = np.tile(b_loc, (n_rx, 1))
 
             # Compute distances
-            AB.append(np.sqrt(np.sum((A - B) ** 2.0, axis=1)))
-            MN.append(np.sqrt(np.sum((M - N) ** 2.0, axis=1)))
-            AM.append(np.sqrt(np.sum((A - M) ** 2.0, axis=1)))
-            AN.append(np.sqrt(np.sum((A - N) ** 2.0, axis=1)))
-            BM.append(np.sqrt(np.sum((B - M) ** 2.0, axis=1)))
-            BN.append(np.sqrt(np.sum((B - N) ** 2.0, axis=1)))
+            AB.append(np.linalg.norm(A - B, axis=1))
```

Comment:
nice. simpler.

Reply:
Yes I myself have written plenty of the hypot calculations over the years and it was a head slap moment when found that numpy provided a function already.